python/ray/experimental/serve/constants.py

```diff
@@ -0,0 +1,2 @@
+#: The interval which http server refreshes its routing table
+HTTP_ROUTER_CHECKER_INTERVAL_S = 2
```

Comment:
Do you want people to be able to change this?

Reply:
This can be modified by changing the global variable before `serve.init`